Create Meeting 97.md (#329)
ethereum-cat-herders/PM: Meeting 97.md

# Ethereum Cat Herders Meeting #97 Notes
### Meeting Date/Time: 2022-Nov-9-12:00-UTC
### Meeting Duration: 34 minutes
### [Audio/Video of the meeting:](https://www.youtube.com/watch?v=cdEcRObZTUY)
### Moderator: Pooja Ranjan
### Notes: HenryJK4

> ### Summary: Ethereum Cat Herders Meeting #97 Notes

Reviewer: ```suggestion
### Summary: 
```

Reviewer: Add Summary - Action Items or Decisions made (if any)

> ### Moderator: 

Reviewer: ```suggestion
### Moderator: Pooja Ranjan
```

**Pooja Ranjan:** 
* welcome to ethereum cat herders meeting  #97. I have shared agenda and chat. The first item that it says here's Ethereum network upgrade. 

> We have two more new joiner today. Henry and Elif. Henry and Elif to introduce them selves.

Reviewer: ```suggestion
We have two more new joiner today. Henry and Elif. Henry and Elif to introduce themselves.
```

**Henry:**
* Here to take notes and to help out. If anyone wants to know more just reach out.

**Elif:**
* Technical product manager,  just wants to contribute, and to help improve the Ethereum foundation.

**Pooja Ranjan:** 
* Thank you Henry and Elif for joining. I saw that Henry had filled out application for Ethereum Cat Herders. I will add you to the Notes schedule and providing me with access to not Channel and to be part of the calendar. Hopefully starting from the next cat herders meeting and from there making adjustments based on my availability. Thank you to Henry and thank you to Elif. Elif I have seen you in the ??meetings and its very nice to have you here and have some of your expertise that you bring to the table.

**Elif:**
* Thank you. 

> # 1. Agenda Item1: Ethereum network upgrades

Reviewer: ```suggestion
# 1. Ethereum network upgrades
```

**Pooja Ranjan:** 

> * As we all know Shanghai is the next plan ?? that is expected on Ethereum blockchain. Ethereum cat herders have updated a page for Shanghai  There's a dedicated page where people can find out what's up in Shanghai. What proposals are there and if there's any important meetings going on that they might be interested to follow along. The upgrade is expected sometime* in 2023. We have updated the page with CFI proposals, US breakout recording that happened last Friday. The upgrade checklist is also available at ethereumcatherders.com/shanghai_upgrade . If you have any resources or information that you'd like to be added to that page. You can create an issue there or create a ??  We just want our community to know well in advance what is expected with Shanghai any kind of confusion ?? any network upgrade. As I mentioned about EF ?? breakout room. This meeting happened last Friday. There is discussion around EVM proposals to be considered for Shanghai upgrade. The general agreement was to have all or none of the EVM on the next upgrade. The team epsilon has proposed 5 proposals which are there to improve the current EVM and the discussion suggested having all or none in order to avoid any kind of changes at the last minute. ?? suggested a test now names “Shangdown”* Shangdown* is a test net started by EtheruemGSteam* end it is proposed for Shanghai proposals for now. The version one contained we EVM proposals because just trying to check out the proposals that were there under CFI.  However ?? is out now and the shanghai.eth.org.io Shanghai has been rebooted on Sunday with a new ID. The content for test net has been shared with the US meeting on Friday and we have a list. According to the list the Shanghai core contains 4 proposals the EIP3651, EIP3855, EIP3860, EIP4895 . However we are expecting the version two with 4 proposals EIP3540, EIP3670, EIP4200, EIP4750 . There is another that is up for discussion and that is a big proposal EIP 4844 Some people are expecting to have a separate test net for that is going to have both proposals and education there providing that will have its own separate test net. Talking about test Nets there are a couple of test nuts under discussion at this moment. Goerli and Roxton*. Team Goerli had a committee meeting last week and it was further discussed in the all core Dev.  And consensus layer meeting. The present issue with Goerli is that it will have the test ether it seems some people are trying to trade it. There are people showing up on Discord asking about trading options of Eth. Goerli which is not right. We don't believe that test Eth. should be treated. The general recommendation is please do not buy it. ?? recommended to migrate to the Sipolia and we see that Sipolia is there for the longer-term and we hoping to have further discussion on the up coming all core dev meeting this Thursday. Similarly we hope to have a discussion around the date for rubston Sunset so watch out for the next meeting which is planned on Thursday at 1400 UTC. We have also added the link to specs on ELN ?? you can find it on Shanghai and ?? respectively. Any question so far? I know this is just general update. OK in that case maybe we can move on. 

Reviewer: ```suggestion
* As we all know Shanghai is the next plan upgrade that is expected on Ethereum blockchain. Ethereum cat herders have updated a page for Shanghai  There's a dedicated page where people can find out what's up in Shanghai. What proposals are there and if there's any important meetings going on that they might be interested to follow along. The upgrade is expected sometime* in 2023. We have updated the page with CFI proposals, US breakout recording that happened last Friday. The upgrade checklist is also available at ethereumcatherders.com/shanghai_upgrade . If you have any resources or information that you'd like to be added to that page. You can create an issue there or create a ??  We just want our community to know well in advance what is expected with Shanghai any kind of confusion ?? any network upgrade. As I mentioned about EF ?? breakout room. This meeting happened last Friday. There is discussion around EVM proposals to be considered for Shanghai upgrade. The general agreement was to have all or none of the EVM on the next upgrade. The team epsilon has proposed 5 proposals which are there to improve the current EVM and the discussion suggested having all or none in order to avoid any kind of changes at the last minute. ?? suggested a test now names “Shangdown”* Shangdown* is a test net started by EtheruemGSteam* end it is proposed for Shanghai proposals for now. The version one contained we EVM proposals because just trying to check out the proposals that were there under CFI.  However ?? is out now and the shanghai.eth.org.io Shanghai has been rebooted on Sunday with a new ID. The content for test net has been shared with the US meeting on Friday and we have a list. According to the list the Shanghai core contains 4 proposals the EIP3651, EIP3855, EIP3860, EIP4895 . However we are expecting the version two with 4 proposals EIP3540, EIP3670, EIP4200, EIP4750 . There is another that is up for discussion and that is a big proposal EIP 4844 Some people are expecting to have a separate test net for that is going to have both proposals and education there providing that will have its own separate test net. Talking about test Nets there are a couple of test nuts under discussion at this moment. Goerli and Roxton*. Team Goerli had a committee meeting last week and it was further discussed in the all core Dev.  And consensus layer meeting. The present issue with Goerli is that it will have the test ether it seems some people are trying to trade it. There are people showing up on Discord asking about trading options of Eth. Goerli which is not right. We don't believe that test Eth. should be treated. The general recommendation is please do not buy it. ?? recommended to migrate to the Sipolia and we see that Sipolia is there for the longer-term and we hoping to have further discussion on the up coming all core dev meeting this Thursday. Similarly we hope to have a discussion around the date for rubston Sunset so watch out for the next meeting which is planned on Thursday at 1400 UTC. We have also added the link to specs on ELN ?? you can find it on Shanghai and ?? respectively. Any question so far? I know this is just general update. OK in that case maybe we can move on. 
```

Reviewer: A lot of missing terms from the notes. Add timestamp for people to get to the conversation or identity the word.

> # 2. Agenda Item2: Events and hackathon

Reviewer: ```suggestion
# 2. Events and hackathon
```

**Pooja Ranjan:** 

> * I have 2 hackathons here eth India and eth Denver. If you want to know about more hackathons we do have a Channel on Ethereum cat herders discord please share any hackathon event that you're aware of Will be happy to add it over here and let the community know how to participate and if you're planning to join. Moving on the next one here is about sponsoring events and hackathons. I have been considering like we get to know many people here many people join Ethereum cat herders discord to find a way to be able to contribute to Ethereum community we are trying to share information every time we get the opportunity to but I have been considering expanding the support sponsoring events with small prizes. For implementing any proposal under discussion. I would like to hear thoughts of people joined here. Like what do they think about encouraging people to maybe be able to use ?? in hackathons and all. Are they open to cut herders providing some small bounty for these kind of implementations. I don't see any objection here, it seems. ?? team. Actually got in touch with the eth Global team and came across the template by which they providing this opportunity to many … in the ecosystem they can be able to encouraged hackathon participants to be useing their product similarly we would like to encourage hackathon participants to be using EIP Core interface 

Reviewer: ```suggestion
* I have 2 hackathons added here -  ETHIndia and ETHDenver. If you want to know about more hackathons, we do have a Channel on Ethereum cat herders discord. please share any hackathon event that you're aware of . We will be happy to add it over here and let the community know how to participate and if you're planning to join. Moving on the next one here is about sponsoring events and hackathons. I have been considering like we get to know many people here many people join Ethereum cat herders discord to find a way to be able to contribute to Ethereum community we are trying to share information every time we get the opportunity to but I have been considering expanding the support sponsoring events with small prizes. For implementing any proposal under discussion. I would like to hear thoughts of people joined here. Like what do they think about encouraging people to maybe be able to use EIPs in hackathons and all. Are they open to cut herders providing some small bounty for these kind of implementations. I don't see any objection here.  Actually, I got in touch with the ETHGlobal team and came across the template by which they providing this opportunity to many projects in the ecosystem they can be able to encouraged hackathon participants to be using their product similarly we would like to encourage hackathon participants to be using EIP Core interface 
```
